src/app/[id]/page.tsx

```diff
@@ -34,7 +42,7 @@ import { useBeforeUnload } from '@/utils/hooks/useBeforeUnload';
 import { formSteps } from '@/utils/constants/formSteps';
 
 type Inputs = z.infer<typeof FormDataSchema>;
-type Service = z.infer<typeof ServiceSchema>;
+export type Service = z.infer<typeof ServiceSchema>;
```

Comment:
would probably be better to recreate the Service type in the ServicesReview component instead of exporting it from the form page